lxl-web/src/lib/contexts/matomo.ts

```diff
@@ -30,6 +30,8 @@ export function setMatomoTracker() {
 	if (initializedMatomoTracker) {
 		tracker.set(initializedMatomoTracker);
 		console.info('Matomo tracker set');
+	} else {
+		console.info('Matomo is disabled');
```

Comment:
I think the console info log could have a slightly more specific message (e.g. `Missing Matomo id`) telling why it is disabled.

Reply:
> Or maybe no console log at all when it's not set (to prevent having too much information to sift out)?

This will hopefully not happen in production?

> Users could maybe also become unnecessarily annoyed by the Matomo tracker set message

Users don't look at the console?

But ok, this was more of a debugger thing when Matomo implementation was shaky. Seems to be working now, so I don't feel very strongly about it either way :)